scripts/test_execution_tutorial.sh

```diff
@@ -0,0 +1,20 @@
+set -e # first error should stop execution of this script
+
+# Identify the bin dir in the distribution, and source the common include script
+BASE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/.. && pwd )"
+source ${BASE_DIR}/scripts/common.sh
+
+# also trap errors, to reenable terminal settings
+trap onExit ERR
+export CLASSPATH=tutorial/execution-tutorial/target/execution-tutorial-assembly-0.13.1.jar
+time $SCALD \
```

Comment:
can we not run with scald.rb? A main goal here is to just run like a normal JVM app. Why can't we run this using `java jar`? That should be the full demo, in my view.